docs/docs/cmd/spo/file/file-roleassignment-add.mdx

```diff
@@ -26,10 +26,16 @@ m365 spo file roleassignment add [options]
 : The SharePoint Id of the principal. It may be either a user id or group id to add a role assignment for. Specify either `upn`, `groupName`, or `principalId` but not multiple.
 
 `--upn [upn]`
-: The upn/email of user to assign role to. Specify either `upn`, `groupName`, or `principalId` but not multiple.
+: The upn/email of user to assign role to. Specify either `upn`, `groupName`, entraGroupId, entraGroupName, or `principalId` but not multiple.
```

Comment:
we should wrap the option names in `` in the description.

```suggestion
: The upn/email of user to assign role to. Specify either `upn`, `groupName`, `entraGroupId`, `entraGroupName`, or `principalId` but not multiple.
```

This also applies to all other places below